go/cmd/dolt/commands/sqlserver/server_test.go

```diff
@@ -187,7 +187,7 @@ func TestServerGoodParams(t *testing.T) {
 
 func TestServerSelect(t *testing.T) {
 	env := dtestutils.CreateEnvWithSeedData(t)
-	serverConfig := DefaultServerConfig().withLogLevel(LogLevel_Fatal).WithPort(15300)
+	serverConfig := DefaultServerConfig().withLogLevel(LogLevel_Fatal).WithPort(15300).WithHost("127.0.0.1")
```

Comment:
Why did we need to add the 127.0.0.1 localhost address here?

Reply:
With no host defined, it will start local server with the new implementation which uses unix socket so there is no specific port being used that it defines here. So, the clients cannot connect using this port number to the server that started as local host.

Reply:
I think I misunderstood here, the docs says if port defined, it should be tcp connection, I'm fixing according to that